roles/rundeck/templates/realm.properties.j2

```diff
@@ -8,6 +6,6 @@
 # Passwords may be clear text, obfuscated or checksummed.  The class
 # org.mortbay.util.Password should be used to generate obfuscated
 # passwords or password checksums
-#
-
-{{ global_pmm_username }}: BCRYPT:{{ global_pmm_password | password_hash('blowfish') }},user,admin
+{% for user in rundeck_manage_users %}
+{{ user.username }}: {{ user.password }},{{ user.role }}
```

Comment:
Why is this plaintext? There are other options until bcrypt is available. That said, you should verify that whatever is generated does indeed match how `org.mortbay.util.Password` would generate them

Reply:
```WARNING

It is recommended to use BCRYPT encrypted passwords with realm.properties as it is the most secure option available. Avoid plain, MD5, and CRYPT.
```

```Warning

The use of CRYPT comes with limitations. Only the first 8 characters of the provided password will be validated when authenticating. This encryption scheme should be avoided if possible.
```

That leaves md5 and plaintext...I'd say they are equal

Reply:
Anything is better than plaintext :)

Reply:
Blowfish hashing enabled now that Ansible can be packed with `passlib`